warehouse/models/staging/payments/elavon/int_elavon__billing_transactions_deduped.sql

```diff
@@ -0,0 +1,73 @@
+{{ config(materialized='table') }}
+
+WITH
+
+billing_transactions AS (
+
+  SELECT * FROM {{ ref('stg_elavon__transactions') }}
+  WHERE batch_type = 'B'
+
+),
+
+dedup_billing_transactions AS (
+
+  SELECT *
+  FROM billing_transactions
+  QUALIFY DENSE_RANK() OVER (ORDER BY execution_ts DESC) = 1
+
+),
+
+int_elavon__billing_transactions_deduped AS (
+
+  SELECT
+
+      payment_reference,
+      payment_date,
+      account_number,
+      routing_number,
+      fund_amt,
+      batch_reference,
+      batch_type,
+      customer_batch_reference,
+      customer_name,
+      merchant_number,
+      external_mid,
+      store_number,
+      chain,
+      batch_amt,
+      amount,
+      surchg_amount,
+      convnce_amt,
+      card_type,
+      charge_type,
+      charge_type_description,
+      card_plan,
+      card_no,
+      chk_num,
+      transaction_date,
```

Comment:
For billing transactions, I would not include the columns that are always null-- Things like `purch_id`, `trn_ref_num`, etc. (the transaction-level data that is not present for billing rows)